core/embed/rust/src/ui/model_mercury/flow/confirm_output.rs

```diff
@@ -212,40 +211,29 @@ fn get_cancel_page(
 }
 
 pub fn new_confirm_output(
-    title: Option<TString<'static>>,
-    subtitle: Option<TString<'static>>,
+    main_params: ConfirmBlobParams,
```

Comment:
Huge struct with most members unused is also not ideal, but I suppose we can live with it if it doesn't cause stack overflows. This is probably heading in the direction of builder pattern / `ConfirmOutputParams`.

Reply:
Definitely, I'd like to put more thought into structuring these params. But felt this is a good first step as the flows creation is bloated even without arg parsing.